docker-compose.yml

```diff
@@ -192,18 +192,18 @@ services:
       - .charon/cluster/node1/validator_keys:/opt/charon/validator_keys
       - ./teku:/opt/charon/teku
 
-  vc2-lighthouse:
-    image: sigp/lighthouse:${LIGHTHOUSE_VERSION:-v3.4.0}
-    entrypoint: /opt/lighthouse/run.sh
+  vc2-nimbus:
+    image: statusim/nimbus-validator-client:${NIMBUS_VERSION:-multiarch-v23.1.1}
     networks: [ cluster ]
     depends_on: [ node2 ]
     restart: unless-stopped
-    environment:
-      LIGHTHOUSE_BEACON_NODE_ADDRESS: http://node2:3600
-      ETH2_NETWORK: ${ETH2_NETWORK:-goerli}
+    command: |
+      --beacon-node=http://node2:3600
+      --secrets-dir=/home/user/validator_keys
+      --validators-dir=/home/user/validator_keys
     volumes:
-      - ./lighthouse/run.sh:/opt/lighthouse/run.sh
-      - .charon/cluster/node2/validator_keys:/opt/charon/keys
+      - .charon/cluster/node2/validator_keys:/home/user/validator_keys
+      - ./nimbus:/home/user/data
```

Comment:
Mounting to `/home/user/data` as recommended by docs - https://nimbus.guide/docker.html
<img width="715" alt="Screenshot 2023-02-06 at 23 15 53" src="https://user-images.githubusercontent.com/43061995/217046349-fb1f73c3-b412-4fab-8858-c5d991862ed7.png">